cmd/snap/cmd_quota.go

```diff
@@ -45,24 +42,45 @@ The quotas command shows all quota groups.
 `)
 
 var shortRemoveQuotaHelp = i18n.G("Remove quota group")
-var longRemoveQuotaHelp = i18n.G(`The remove-quota command removes the given quota group.`)
+var longRemoveQuotaHelp = i18n.G(`
+The remove-quota command removes the given quota group.
 
-type cmdQuota struct {
-	clientMixin
-	// MaxMemory and MemoryMax are mutually exclusive and provided for
-	// convienience, they mean the same.
-	MaxMemory  string `long:"max-memory" optional:"true"`
-	MemoryMax  string `long:"memory-max" optional:"true"`
-	Parent     string `long:"parent" optional:"true"`
-	Positional struct {
-		GroupName string              `positional-arg-name:"<group-name>" required:"true"`
-		Snaps     []installedSnapName `positional-arg-name:"<snap>" optional:"true"`
-	} `positional-args:"yes"`
-}
+Quota groups with sub-groups 
```

Comment:
is this a strange left over?

Reply:
Ah yes I think I messed this up when I was changing the other help messages, this is indeed wrong and should be actually written this time